ui/raidboss/raidboss.js

```diff
@@ -40,4 +40,20 @@ gTimelineController = new TimelineController(Options, new TimelineUI(Options));
 gPopupText = new PopupText(Options);
 // Connect the timelines to the popup text.
 gTimelineController.SetPopupTextInterface(new PopupTextGenerator(gPopupText));
-gPopupText.SetTimelineLoader(new TimelineLoader(gTimelineController));
\ No newline at end of file
+gPopupText.SetTimelineLoader(new TimelineLoader(gTimelineController));
+OverlayPluginApi.overlayMessage(OverlayPluginApi.overlayName, JSON.stringify({'getUserLocation': true}));
+document.addEventListener("onSendUserConfigLocation", function(e) {
+  var jsLoc = (e.detail.location);
+  var cssLoc = jsLoc.replace('.js', '.css');
+  /*load CSS*/
+  var userCSS = document.createElement('link');
+  userCSS.setAttribute('rel', 'stylesheet');
+  userCSS.setAttribute('type', 'text/css');
+  userCSS.setAttribute('href', cssLoc);
+  document.getElementsByTagName('head')[0].appendChild(userCSS);
```

Comment:
I'm a really bad web developer.  Does this work prior to the document load event? I know other elements don't exist, but I am not sure about the head element.  Just curious if there is a race here.

Reply:
no race, the head element is loaded at the moment it parses the head opening tag.  As the parser moves down through the objects within the head element its appending them to the Head element already within the Document Object Model.